doc/m4/FS22-Group-36-M4-Report.md

```diff
@@ -0,0 +1,127 @@
+many points as possible during the game. Every player plays for himself, 
+there are no teams. The game is played with a “Swiss Jass” card set of 36 cards with four suits.
+
+The game is divided into 9 matches (Runden) with changing number of distributed cards per player 
+(5,4,3,2,1,2,3,4,5), and for every match points are distributed.
+
+For the first match, every player gets 5 cards. 
+Then one player after the other must announce how many tricks ("Stich", according to https://en.wikipedia.org/wiki/Jass) 
+they will make in this match, 
+starting from the player right from the current dealer. 
+The last one announcing the number of his tricks for this match must announce several tricks such that the total 
+sum of announced tricks is not equal to the number of cards distributed per player in this match.
+
+
+When the trick announcing round is completed, the game starts. The player to the right of the current dealer starts. 
+The suits don’t matter in this game, just the rank, and there is no trump. The player who played the highest 
+rank in a match, i.e. wins the trick and has to deal the first card for the next trick.
+If there are 2 or more highest ranks in the trick, the next trick stacks on top of the current. In this case, 
+the player who started the current trick has to start the next one too. If the last trick was stacked, the players 
+who played the highest cards draw another card and continue the next trick with these. This may continue until 
+there is only one highest rank in the trick.
+
+When all the tricks in the match are played out, the players count the number of their tricks.
+
+
+Points counting rules:\
+If a player announced the number of tricks correctly, then he gets the number of announced tricks squared as 
+positive points. Otherwise, the player gets the difference as negative points.
+
+Then the cards are distributed for the next match. The dealer duty moves one player to the right, the number 
+of cards that is distributed follows the sequence [5,4,3,2,1,2,3,4,5].
+
+For the match with only 1 card there is a special rule. The players don’t see their own card but put their card 
+with the front revealed on their forehead, so only the other players can see the card, but not the player himself. 
+After the 9 matches, the player with the highest score wins.
+
+## 2. Game view
+The order of the screenshots corresponds to the flow of the game, if possible. Matches 4, 3, 2 and the last match 
+are not listed. The only difference to the five-card match is the number of cards.
+Depends on the result of the game, after the last match, only the winner can see a screw your neighbor cup, 
+well-designed by Carmen. 
+
+### X Technologies
+#### X.x Spring Data REST
+
+#### X.x Hibernate, JPA
+
+#### X.x Jitsi API
+
+#### X.x Mockito
+
+#### X.x Git
+
+
+### X.x High-level components
+#### UI
+
+#### Database
+
+#### Game Logic (side effects)
+
+#### Game Validations
+
+#### Security
+
```

Comment:
Not sure, if I understood the term _component_ i.e. the task _Identify your project’s 3-5 main components._ in the right way. Maybe they mean with components also parts of the application that implement a specific logic, e.g. the stacking rule.